Stripe/STPSectionHeaderView.m

```diff
@@ -24,22 +24,25 @@ + (instancetype)addressHeaderWithConfiguration:(STPPaymentConfiguration *)config
                               addressViewModel:(STPAddressViewModel *)addressViewModel
                                shippingAddress:(STPAddress *)shippingAddress {
     STPSectionHeaderView *headerView = [STPSectionHeaderView new];
-    headerView.title = STPLocalizedString(@"Billing Address", @"Title for billing address entry section");
     switch (config.shippingType) {
         case STPShippingTypeShipping:
-            [headerView.button setTitle:STPLocalizedString(@"Use Shipping", @"Button to fill billing address from shipping address.")
+            [headerView.button setTitle:STPLocalizedString(@"USE SHIPPING", @"Button to fill billing address from shipping address.")
                                       forState:UIControlStateNormal];
             break;
         case STPShippingTypeDelivery:
-            [headerView.button setTitle:STPLocalizedString(@"Use Delivery", @"Button to fill billing address from delivery address.")
+            [headerView.button setTitle:STPLocalizedString(@"USE DELIVERY", @"Button to fill billing address from delivery address.")
                                       forState:UIControlStateNormal];
             break;
     }
     BOOL needsAddress = !addressViewModel.isValid;
     switch (sourceType) {
         case STPSourceTypeCard:
+            headerView.title = STPLocalizedString(@"BILLING ADDRESS", @"Title for billing address entry section");
             needsAddress = needsAddress && config.requiredBillingAddressFields != STPBillingAddressFieldsNone;
             break;
+        case STPSourceTypeSEPADebit:
+            headerView.title = STPLocalizedString(@"ACCOUNT HOLDER ADDRESS", @"Title for address entry section");
+            break;
         default:
```

Comment:
Prob shouldn't use `default` here so that we'll get compiler warnings when more types are added that may be relevant.